tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -302,7 +302,9 @@ private int reportOpenedDescriptors() {
 }
 
 private Browser createBrowser(Shell s, int flags) {
+	// TODO Reduce to reasonable value
 	long maximumBrowserCreationMilliseconds = 90_000;
+	System.setProperty("org.eclipse.swt.internal.win32.Edge.timeout", Long.toString(maximumBrowserCreationMilliseconds));
```

Comment:
```suggestion
	System.setProperty("org.eclipse.swt.internal.win32.Edge.timeout", Long.toString(TimeUnit.SECONDS.toMilllis(90));
```

Reply:
I have moved this into properly typed, static constants and according initialization.